charts/renku/values.yaml

```diff
@@ -540,18 +547,26 @@ notebooks:
         #        secretKeyRef:
         #          name: renku-jupyterhub-postgres
         #          key: jupyterhub-postgres-password
-    
+
     proxy:
       ## Must be set explicitly, generate using `openssl rand -hex 32`
-      secretToken: 
+      secretToken:
 
-    ## Simple default setup for gitlab oauth
-    # auth:
+    ## Configuration for gitlab oauth
```

Comment:
maybe worth explaining here which gitlab oauth this is referring to?

Reply:
Adapted the wording.